netgraph/policy.go

```diff
@@ -46,40 +46,44 @@ func (p Int32Slice) Len() int           { return len(p) }
 func (p Int32Slice) Less(i, j int) bool { return p[i] < p[j] }
 func (p Int32Slice) Swap(i, j int)      { p[i], p[j] = p[j], p[i] }
 
+func (b Bucket) Hash() uint64 {
+	return hrw.Hash([]byte(b.Key + b.Value))
+}
+
 // FindGraph returns random subgraph, corresponding to specified placement rule.
-func (b *Bucket) FindGraph(rnd *rand.Rand, ss ...Selector) (c *Bucket) {
+func (b *Bucket) FindGraph(pivot []byte, ss ...Selector) (c *Bucket) {
```

Comment:
Maybe provide generic interface with `Shuffle` or `Sort` method, instead of `[]byte`?
So that we can use both Random, HRW and 'no shuffle' when needed.